utils.py

```diff
@@ -1,43 +1,59 @@
 #!/usr/bin/env python
 
 import sys
+import array
 import pygame
 
 import wx
 wx.App()
 
 import numpy as np
 
+from PIL import Image
+
 from skimage.color import rgb2gray
 from skimage.transform import resize
 from skimage.io import imread
+from skimage.util import img_as_float
 
 import matplotlib.pyplot as plt
 import matplotlib.image as mpimg
 
+SRC_W = 615 # 640
+SRC_H = 480
+SRC_D = 3
+OFFSET_X = 0 # 400
+OFFSET_Y = 0 # 240
```

Comment:
ah gotcha. It would be great to figure out how to control where the emulator goes and what size it is so we can standardise this.